src/Error/Debugger.php

```diff
@@ -32,6 +33,16 @@
  */
 class Debugger
 {
+    use InstanceConfigTrait;
```

Comment:
What about something like

```php
// Generic config, could be used to refactor other config methods later
Debugger::config('outputMask', [..]);

// Specific method
Debugger::outputMask([...])
Debugger::setOutputMask([...])
```

I don't expect the singleton to be removed right now, but its something I'd like to do/see done in a future release.